ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.12-nightly-x86.yaml

```diff
@@ -26,53 +26,37 @@ tests:
   steps:
     cluster_profile: gcp-qe
```

Comment:
the cron on this job might be running too much, we could lower or create a separate task to lower. Not sure if we need these (and 4.13 gcp) to run once a week; in my opinion could be less

Reply:
Since this is 4.13 on GCP, we could taper it down to once weekly, perhaps - will bring this up in our weekly discussions.